tsconfig.json

```diff
@@ -24,5 +24,6 @@
     "forceConsistentCasingInFileNames": true,
     "esModuleInterop": true,
     "resolveJsonModule": true
-  }
+  },
+  "include": ["src/**/*.ts", "src/**/*.tsx"]
```

Comment:
I think we can just use `"include": ["src"]`

Reply:
Yes it would work too with `"src/**/*"`, I just put the standard ts/tsx blob because it's common practice